src/Microsoft.TestPlatform.PlatformAbstractions/Interfaces/System/IProcessHelper.cs

```diff
@@ -42,6 +42,18 @@ public interface IProcessHelper
         string GetTestEngineDirectory();
 
         /// <summary>
+        /// Gets the location of msdia140.dll, depending on current process architecture..
```

Comment:
nit: may not hardcode name of a consumed assembly here. It could be anything, right?

Reply:
msdia, is just an example here.

Reply:
Move it to `remarks` section in that case.

Reply:
resolved